entity-registry/src/main/java/com/linkedin/metadata/aspect/patch/template/dashboard/DashboardInfoTemplate.java

```diff
@@ -74,6 +75,12 @@ public JsonNode transformFields(JsonNode baseNode) {
             DATASET_EDGES_FIELD_NAME,
             Collections.singletonList(DESTINATION_URN_FIELD_NAME));
 
+    transformedNode =
```

Comment:
If there isn't already a test for this let's create one, or if there is add in mapping this field

Reply:
I had already added a test for `DashboardInfoTemplate` in https://github.com/datahub-project/datahub/pull/12433/files#diff-b6808c495535fbf32144cbc4e6f16eb762b55c377c63ef929758d21760c28437

The test operates on `applyPatch` method, which goes through both `transformFields` and `rebaseFields` methods, as can be seen in https://github.com/datahub-project/datahub/blob/47134c272bd82ff8d00b6a30c725fbde4165335c/entity-registry/src/main/java/com/linkedin/metadata/aspect/patch/template/Template.java#L51-L86